QATCH/ui/mainWindow.py

```
@@ -9032,4 +9032,4 @@ def render_frame(self, time_w, freq_w, diss_w, stats: dict, thresholds: dict):
 
     def close(self):
         plt.ioff()
-        plt.close(self.fig)
\ No newline at end of file
+        plt.close(self.fig)
```

Ruff (0.15.15)

[error] Undefined name `plt`

(F821)